bowler.js

```diff
@@ -0,0 +1,247 @@
+
+const Frame = require('./frame');
+const Roll = require('./roll');
+
+class Bowler{   
+
+  constructor(id, name, frames, bowlerTotal, strikeMsg, spareMsg, noScoreMsg){
```

Comment:
I like the idea of being able to set up your own name and personalised messages!